src/Microsoft.SqlTools.ServiceLayer/QueryExecution/DataStorage/ServiceBufferFileStreamReader.cs

```diff
@@ -147,8 +154,9 @@ public IList<DbCellValue> ReadRow(long fileOffset, IEnumerable<DbColumnWrapper>
         /// Reads a short from the file at the offset provided
         /// </summary>
         /// <param name="fileOffset">Offset into the file to read the short from</param>
+        /// <param name="col">Not used</param>
         /// <returns>A short</returns>
-        public FileStreamReadResult ReadInt16(long fileOffset)
+        public FileStreamReadResult ReadInt16(long fileOffset, DbColumnWrapper col)
```

Comment:
This is mostly to get all the signatures to be the same, that way we can utilize the "method group" style of assigning delegates. I'm not thrilled with the idea either, so I'll go back and tweak it.